### DISCUSSION TITLE
Are webhooks still in alpha?

### DISCUSSION THREAD

**Discussion by kgalang** [supabase/supabase#11705] [Questions] [answered]
Hi, I'm still seeing that webhooks are in alpha. Is it generally stable for simple triggers like `insert user` -> `POST call`?

**Comment by steve-chavez** [comment, marked as answer]
Hello,

> Is it generally stable for simple triggers like insert user -> POST call?

It's getting close to stable. We just released a new [pg_net version](https://github.com/supabase/pg_net/releases/tag/v0.7)(underlying extension for webhooks).

Make sure you create a new project or pause/unpause an existing one to have the 0.7 version, this can be checked with:

```sql
select extversion from pg_extension where extname = 'pg_net';
```
